agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -1122,442 +1149,543 @@
-							"listen": "prerequest",
-							"script": {
-								"exec": [],
-								"type": "text/javascript"
-							}
-						}
-					],
-					"request": {
-						"method": "GET",
-						"header": [],
-						"url": {
-							"raw": "{{HOST}}/v0/services",
-							"host": [
-								"{{HOST}}"
-							],
-							"path": [
-								"v0",
-								"services"
-							]
-						}
-					},
-					"response": []
-				},
-				{
-					"name": "Service by ID",
-					"request": {
-						"method": "GET",
-						"header": [],
-						"url": {
-							"raw": "{{HOST}}/v0/services/{{SERVICE_ID}}",
-							"host": [
-								"{{HOST}}"
-							],
-							"path": [
-								"v0",
-								"services",
-								"{{SERVICE_ID}}"
-							]
+							"name": "Linked Verifiable Presentation Service",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{HOST}}/v0/services/linked-verifiable-presentation-service",
+									"host": [
+										"{{HOST}}"
+									],
+									"path": [
+										"v0",
+										"services",
+										"linked-verifiable-presentation-service"
+									]
+								}
+							},
+							"response": []
 						}
-					},
-					"response": []
+					]
 				},
 				{
-					"name": "Linked Verifiable Presentation Service",
-					"request": {
-						"method": "GET",
-						"header": [],
-						"url": {
-							"raw": "{{HOST}}/v0/services/linked-verifiable-presentation-service",
-							"host": [
-								"{{HOST}}"
+					"name": "Documents",
+					"item": [
+						{
+							"name": "List all Documents",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"const jsonData = JSON.parse(responseBody);",
+											"",
+											"if (Array.isArray(jsonData) && jsonData.length > 0) {",
+											"    const lastItem = jsonData[jsonData.length - 1];",
+											"",
+											"    if (lastItem && typeof lastItem === 'object') {",
+											"        const documentId = lastItem.id;",
+											"",
+											"        if (documentId) {",
+											"            pm.collectionVariables.set(\"DOCUMENT_ID\", documentId);",
+											"        }",
+											"    }",
+											"}",
+											""
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
 							],
-							"path": [
-								"v0",
-								"services",
-								"linked-verifiable-presentation-service"
-							]
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{HOST}}/v0/documents",
+									"host": [
+										"{{HOST}}"
+									],
+									"path": [
+										"v0",
+										"documents"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Document by ID",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{HOST}}/v0/documents/4cf0f658-0713-4c19-a7ac-82583381ec47",
```

Comment:
`4cf0f658-0713-4c19-a7ac-82583381ec47` should be replaced with `{{DOCUMENT_ID}}` :)